modules/migration-mtc-release-notes-1-7-01.adoc

```diff
@@ -22,7 +22,7 @@ MigPlan cannot be validated because the destination namespace starts with a non-
 The Cloud propagation phase in the migration controller is not functioning due to missing labels on Velero pods. The `EnsureCloudSecretPropagated` phase in the migration controller waits until replication repository secrets are propagated on both sides. As this label is missing on Velero pods, the phase is not functioning as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088026[*BZ#2088026*])
 
 .Default CPU requests on Velero/Restic are too demanding when making scheduling fail in certain environments
-Default CPU requests on Velero/Restic are too demanding when making scheduling fail in certain environments. Default CPU requests for Velero and Restic Pods are set to 500m. These values are high. The resources can be configured in DPA using the `podConfig` field for Velero and Restic. Migration operator should set CPU requests to a lower value, such as 100m, so that Velero and Restic pods can be scheduled in resource constrained environments MTC often operates in. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088022[*BZ#2088022*])
+Default CPU requests on Velero/Restic are too demanding when making scheduling fail in certain environments. Default CPU requests for Velero and Restic Pods are set to 500m. These values are high. The resources can be configured in DPA using the `podConfig` field for Velero and Restic. Migration operator should set CPU requests to a lower value, such as 100m, so that Velero and Restic pods can be scheduled in resource constrained environments {mtc-short} often operates in. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088022[*BZ#2088022*])
```

Comment:
First non-title mention in module
```suggestion
Default CPU requests on Velero/Restic are too demanding when making scheduling fail in certain environments. Default CPU requests for Velero and Restic Pods are set to 500m. These values are high. The resources can be configured in DPA using the `podConfig` field for Velero and Restic. Migration operator should set CPU requests to a lower value, such as 100m, so that Velero and Restic pods can be scheduled in resource constrained environments {mtc-first} often operates in. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2088022[*BZ#2088022*])
```